perfkitbenchmarker/package_managers.py

```diff
@@ -115,11 +115,39 @@ def GetServiceName(self, package_name):
     """
     pass
 
+  def SetupProxy(self):
+    """
+        This function setup proxy if your cloud is currently installed behind proxy server
+    """
+    env_file = "/etc/environment"
+
+    commands = []
+
+    add_proxy_conf = "sudo touch %s" % env_file
+    commands.append(add_proxy_conf)
+
+    if FLAGS.http_proxy:
+        add_http_proxy = "echo -e 'http_proxy=%s' | sudo tee -a %s" % (FLAGS.http_proxy, env_file)
```

Comment:
Why `-e`? I don't think there's really a need to interpret escape sequences, and doing so may have odd side effects, i.e. spurious newlines if the proxy value contains `\n`.